mindsdb/integrations/handlers/file_handler/file_handler.py

```diff
@@ -101,29 +108,43 @@ def _handle_source(file_path, clean_rows=True, custom_parser=None):
             header, file_data = custom_parser(data, fmt)
             df = pd.DataFrame(file_data, columns=header)
 
-        elif fmt == 'parquet':
+        elif fmt == "parquet":
             df = pd.read_parquet(data)
 
-        elif fmt == 'csv':
+        elif fmt == "csv":
             df = pd.read_csv(data, sep=dialect.delimiter, index_col=False)
 
-        elif fmt in ['xlsx', 'xls']:
+        elif fmt in ["xlsx", "xls"]:
             data.seek(0)
             df = pd.read_excel(data)
 
-        elif fmt == 'json':
+        elif fmt == "json":
             data.seek(0)
             json_doc = json.loads(data.read())
             df = pd.json_normalize(json_doc, max_level=0)
 
+        elif fmt == "txt":
+            from langchain.document_loaders import TextLoader
```

Comment:
if file handler is our base handler we need to add langchain to mindsdb requirements